### PR TITLE
Create Add Membership Command

### PR DESCRIPTION
**Pull request by kab-dot**

### CODE REVIEW

**Comment by kinjalagarwal1810** [comment]
Looks good! Good effort

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -17,25 +18,26 @@
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
+    public static final Membership EMPTY_MEMSHIP = new Membership("T1");
```
Comment (billyhoce) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:21]: Might want to rename this constant to better fit the "T1"!

File: src/test/java/seedu/address/logic/commands/AddMemshipCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+//import static seedu.address.logic.commands.CommandTestUtil.VALID_MSHIP_AMY;
```
Comment (billyhoce) [inline src/test/java/seedu/address/logic/commands/AddMemshipCommandTest.java:5]: Might want to remove these unused imports!

File: src/main/java/seedu/address/logic/commands/AddMemshipCommand.java [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Membership;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a membership tier for a person to the address book.
+ */
+public class AddMemshipCommand extends Command {
```
Comment (billyhoce) [inline src/main/java/seedu/address/logic/commands/AddMemshipCommand.java:20]: Would it be better if the command was changed from `AddMemship` to `ChangeMemship`? Also, might be better to name this class the full name `ChangeMembershipCommand`, but the `COMMAND_WORD` can stay as `changemship`!

File: src/test/java/seedu/address/model/person/PersonTest.java [hunk 4]
```diff
@@ -53,47 +53,38 @@ public void isSamePerson() {
 
     @Test
     public void equals() {
-        // same values -> returns true
```
Comment (billyhoce) [inline src/test/java/seedu/address/model/person/PersonTest.java:56]: Could you maybe share why this was removed?

**Review by billyhoce — COMMENTED** [review]
In general, LGTM! Well written code!

File: src/main/java/seedu/address/logic/commands/AddMemshipCommand.java (hunk 3 above)
Comment (kab-dot) [reply src/main/java/seedu/address/logic/commands/AddMemshipCommand.java:20]: Okay good idea, will do in the next iteration

**Pull request merged**